rllib/connectors/action/pipeline.py

```diff
@@ -28,13 +28,13 @@ def __call__(self, ac_data: ActionConnectorDataType) -> ActionConnectorDataType:
             ac_data = c(ac_data)
         return ac_data
 
-    def to_config(self):
+    def to_state_dict(self):
         return ActionConnectorPipeline.__name__, [
-            c.to_config() for c in self.connectors
+            c.to_state_dict() for c in self.connectors
         ]
```

Comment:
love this.
the only high level comment though is that this may not be a dict, like for example here, the state is a list.
name these to_state() / from_state() ?

Reply:
Got it. changed.